docs/source/nodes_and_pipelines/micro_packaging.md

```diff
@@ -80,10 +80,10 @@ If a micro-package has embedded requirements and a project `requirements.in` fil
 You can pull a micro-package from different locations, including local storage, PyPI and the cloud:
 
 | Operation                      | Command                                                                               |
-| ------------------------------ | ------------------------------------------------------------------------------------- |
-| Pulling from a local directory | `kedro micropkg pull <project-root>/src/dist/<pipeline_name>-0.1-py3-none-any.tar.gz` |
+| ------------------------------ |---------------------------------------------------------------------------------------|
+| Pulling from a local directory | `kedro micropkg pull <project_root>/src/dist/<pipeline_name>-0.1-py3-none-any.tar.gz` |
```

Comment:
```suggestion
| Pulling from a local directory | `kedro micropkg pull dist/<pipeline_name>-0.1-py3-none-any.tar.gz` |
```
`src/dist` should definitely be just `dist` now (out of date documentation), and I _think_ we won't need the full path here (but could be wrong - definitely check that!).

Reply:
I confirmed that a relative path works too. Changes made in [790c9e0](https://github.com/kedro-org/kedro/pull/1753/commits/790c9e0478d228a22d50c0bb3a39bacfa8ef24de).